sdk/resources/Microsoft.Azure.Management.Resource/src/generate.ps1

```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "latest" -AutoRestCodeGenerationFlags "--tag=package-resources-2019-07" -SdkGenerationDirectory "$PSScriptRoot\Generated"
\ No newline at end of file
+Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "latest" -AutoRestCodeGenerationFlags "--tag=package-policy-2019-06" -SdkGenerationDirectory "$PSScriptRoot\Generated"
```

Comment:
>--tag=package-policy-2019-06" [](start = 177, length = 29)

instead of updating the cmd, can we add a new line of command and leave it as commented?